### PR TITLE
scoop: added Bengali translation

### PR DESCRIPTION
**Pull request by siddiq0611**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages.bn/windows/scoop.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+`scoop info {{প্যাকেজ}}`
+
+- একটি প্যাকেজ সন্ধান করুন:
+
+`scoop search {{প্যাকেজ}}`
+
+- সব প্রাচীন সংস্করণ সরানো এবং ডাউনলোড ক্যাশ পরিষ্কার করুন:
+
+`scoop cleanup --cache --all`"
```
Comment (Magrid0) [inline pages.bn/windows/scoop.md:32]:
```suggestion
`scoop cleanup --cache --all`"

```

**Review by Magrid0 — CHANGES_REQUESTED** [review]
You have to add a new line at the end of every page.

**Comment by siddiq0611** [comment]
I have committed the requested changes

**Review by Magrid0 — APPROVED** [review]
Looks good to me, thanks for your contribution!

**Review by kbdharun — COMMENTED** [review]
LGTM, Welcome to tldr and thanks for your contribution.

**Pull request merged**